CHANGELOG.md

```diff
@@ -4,9 +4,6 @@
 
 ## Enhancements
 
-* All Realm annotations are now kept at runtime, allowing runtime tools access to them (#5344).
```

Comment:
@vivekkiran These shouldn't be removed. Apart from that everything looks fine to me